addon/adapters/pouch.js

```diff
@@ -365,9 +365,9 @@ export default DS.RESTAdapter.extend({
 
     //Delete the children
     let deletedChildren = [];
-    snapshot.eachRelationship(function(name, descriptor) {
+    snapshot.eachRelationship((name, descriptor) => {
       if(descriptor.kind === 'hasMany' && descriptor.options['cascadeDestroy']) {
-        let mappedPromise = snapshot.record.get(name).then(function(childRecords) {
+        let mappedPromise = snapshot.record.get(name).then(childRecords => {
```

Comment:
Ha... I do that in multiple places... You do raise a good point here about this repo and consistency. I just created a new issue #149 for a contributing guide. Perhaps preferences like this could be defined in there.